src/main.js

```diff
@@ -115,6 +115,10 @@ var mainTitle = document.querySelector('.poster-title')
 var mainImageURL = document.querySelector('.poster-img')
 var mainQuote = document.querySelector('.poster-quote')
 
+var posterGrid = document.querySelector('.saved-posters-grid')
+var displaySavedPosters = document.createElement('article')
+
+posterGrid.appendChild(displaySavedPosters('h2'))
```

Comment:
Not sure if we'll use this but thanks for writing it out. Maybe we can do it with `.innerHTML` first and then see if we can replace that with an `.appendChild`?